### ISSUE TITLE
Couldn't load module library

### ISSUE THREAD

**Issue by jaooe** [SpencerSharkey/gmsv_mongolink#1]
Getting this error on windows 10 x64, I've got the mongoclient.dll next to my hl2.exe

**Comment by P4sca1** [comment]
Is there any mongodb implementation for garrys mod that is still maintained?

**Comment by SpencerSharkey** [comment]
I don't think so- I'm pretty busy these days but I may be able to patch it up.

Of course, if anyone else wants to take a stab at it... feel free!

**Comment by ghost** [comment]
@P4sca1 What exactly do you need a mongo driver in Garry's mod for? It doesn't seem like the kind of thing that makes a lot of sense in the majority of cases, there may be a better tool for the job.

**Comment by P4sca1** [comment]
@Frappocalypse Im currently using MySQL for my gamemode. The gamemode includes a character system for example which allows other addons to add different attributes (e.g. a level addon adds a level attribute to the character). With mysql and its fixed data structure, it is quite hard to realize this. I am currently using another table which holds the character id, the attribute name and a value field with a type of TEXT, which is then parsed to the correct type in lua. Using mongodb and its document approach would make dynamic and customizable data structures more easy to manage.

**Comment by dhkatz** [comment]
@P4sca1 https://github.com/dhkatz/gmsv_mongodb
